pinc/faq.inc

```diff
@@ -71,8 +71,8 @@ $external_faq_overrides = [
     "request-support" => [
         "en" => "https://www.pgdp.net/wiki/DP_Official_Documentation:Main/Request_Support",
     ],
-    "transliterating-greek" => [
-        "en" => "https://www.pgdp.net/wiki/DP_Official_Documentation:Proofreading/Transliterating_Greek",
+    "greek" => [
+        "en" => "$wiki_url/Greek",
```

Comment:
We need to hard-code the value as `https://www.pgdp.net/wiki/` so that this works for TEST as well as PROD. We can't rely on `$wiki_url` for this.

Reply:
Done.